packages/dev/core/src/Shaders/ShadersInclude/lightsFragmentFunctions.fx

```diff
@@ -154,4 +154,49 @@ vec3 computeProjectionTextureDiffuseLighting(sampler2D projectionLightSampler, m
 	strq /= strq.w;
 	vec3 textureColor = texture2D(projectionLightSampler, strq.xy).rgb;
 	return textureColor;
-}
\ No newline at end of file
+}
+
+#ifdef AREALIGHTUSED
+#include<ltcHelperFunctions>
+
+uniform sampler2D areaLightsLTC1Sampler;
+uniform sampler2D areaLightsLTC2Sampler;
+
+lightingInfo computeAreaLighting(sampler2D ltc1, sampler2D ltc2, vec3 viewDirectionW, vec3 vNormal, vec3 vPosition, vec4 lightData, vec3 halfWidth, vec3 halfHeight, vec3 diffuseColor, vec3 specularColor, float roughness ) 

```

Comment:
Can we add the reference to the paper we used just before the function ?

Reply:
it also looks really similar to computeAreaPreLightingInfo ? nothing we should factorize ? the rest is splitted cause pbr and standard do not do the same.